### ISSUE TITLE
[Exception][WebStorm 2023.2.2] java.util.NoSuchElementException: Collection contains no element matching the predicate.

### ISSUE THREAD

**Issue by petrsimanek** [Codium-ai/codiumai-jetbrains-release#123]
# User Description
_User did not enter any detailed information._

# Stack Trace
```
java.util.NoSuchElementException: Collection contains no element matching the predicate.
	at ai.codium.language.JsTsLanguageService.getPsiIdentifierElement(JsTsLanguageService.kt:181)
	at ai.codium.ui.CodiumGutterIconLineMarkerProvider.collectSlowLineMarkers(CodiumGutterIconLineMarkerProvider.kt:48)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.queryProviders(LineMarkersPass.java:225)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.lambda$doCollectMarkers$2(LineMarkersPass.java:113)
	at com.intellij.codeInsight.daemon.impl.Divider.divideInsideAndOutsideInOneRoot(Divider.java:95)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.doCollectMarkers(LineMarkersPass.java:103)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.doCollectInformation(LineMarkersPass.java:76)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:57)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:390)
	at com.intellij.platform.diagnostic.telemetry.helpers.TraceKt.runWithSpanIgnoreThrows(trace.kt:77)
	at com.intellij.platform.diagnostic.telemetry.helpers.TraceUtil.runWithSpanThrows(TraceUtil.java:24)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:386)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1133)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$3(PassExecutorService.java:377)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:61)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:376)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:352)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:201)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:207)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:350)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:190)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)

```

# Other Information
```
=== About ===
Build version: WebStorm 2023.2.2 Build: #WS-232.9921.42 September 12, 2023
Theme: Dark
JRE: 17.0.8+7-b1000.22, JetBrains s.r.o.
JVM: 17.0.8+7-b1000.22, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Mac OS X 14.0 (aarch64)

=== Plugins ===
Custom plugins: [.ignore (4.5.1), CodiumAI - Integrity Agent powered by GPT-3.5&4 (0.6.10), .env files support (2023.2), GitHub Copilot (1.3.0.3323), Translation (3.5.4), Perl (2023.2), Gradle (232.9921.62), Import Cost (1.4.232), String Manipulation (9.11.3)]
Disabled plugins:[VibrantInk Color Scheme (232.9921.42), Eclipse Keymap (232.9921.42), GitHub (232.9921.42), Terminal for Gateway (232.9921.42), HTML Tools (232.9921.42), Qodana (232.9921.42), Visual Studio Keymap (232.9921.42), Task Management (232.9921.42), WarmNeon Color Scheme (232.9921.42), Performance Testing (232.9921.42), NetBeans Keymap (232.9921.42), Karma (232.9921.42), Performance Testing (Dynamic Plugins) (232.9921.42), Dev Containers (232.9921.42), Cobalt Color Scheme (232.9921.42), Mercurial (232.9921.42), Vagrant (232.9921.42), Polymer & Web Components (232.9921.42), VSCode Keymap (232.9921.42), SSH Remote Run (232.9921.42), Blackboard Color Scheme (232.9921.42), Angular and AngularJS (232.9921.42), OpenAPI Specifications (232.9921.42), TSLint (232.9921.42), GitHub Color Scheme (232.9921.42), Node.js Remote Interpreter (232.9921.42), Remote Development Gateway (232.9921.42), Vue.js (232.9921.42), Monokai Color Scheme (232.9921.42), Docker (232.9921.42), Time Tracking (232.9921.42), Twilight Color Scheme (232.9921.42), Railscasts Color Scheme (232.9921.42), Vite (232.9921.42), Space (232.9921.42), Tailwind CSS (232.9921.42), Subversion (232.9921.42), Webpack (232.9921.42), Shared Project Indexes (232.9921.42), FTP/SFTP/WebDAV Connectivity (232.9921.42), All Hallow's Eve Color Scheme (232.9921.42)]
```

┆Issue is synchronized with this [Trello card](https://trello.com/c/XzHCJtMG) by [Unito](https://www.unito.io)

**Comment by aldanchenko** [comment]
Hi @petrsimanek 

Thanks for sharing! We are currently working on a fix. Should be fixed in the next version. I'll let you know when the next version comes out.

Best regards,
Alex

**Comment by ArtsiomCh** [comment]
Fixed in 0.6.12
